src/components/layout/MobileHeader/index.tsx

```diff
@@ -48,7 +49,7 @@ const MobileHeader: React.FC<IMobileHeaderProps> = ({ title, back, notificationI
           notificationId={notificationId}
         />
       ) : (
-        <Avatar address={userPubkey as `0x${string}`} width="1.875em" height="1.875em" />
+        <Avatar address={getEthChainAddress(userPubkey) as `0x${string}`} width="1.875em" height="1.875em" />
```

Comment:
This bit is non-DRY. Should we add it in the context or something ?

Reply:
I'd rather not have two separate states that basicallty just hold address information